Yoast/wordpress-seo
# Make the scrollable tables work again.

## Summary

This PR can be summarized in the following changelog entry:

* Makes the scrollable tables work in the react tabs.

## Relevant technical choices:

I've tried several things, the main issue here is we'd need a callback after completion of `dangerouslySetInnerHTML` but there isn't such a thing.
We know when the help center mounts, we know when the react tabs mount, there's also an `onSelect` event when a react tabs get selected but... the actual content is injected after that.

Scrollable tables instead need to check the actual width of the table and its parent and this needs to happen when the table is in the DOM. Also on window resize, and orientation change.

I've ended up using a `setTimeout`, not happy for that, open to suggestions.

## Test instructions

This PR can be tested by following these steps:

* on small screens, verify the template variables tables are scrollable. Screenshot:

![screen shot 2017-10-09 at 17 22 23](https://user-images.githubusercontent.com/1682452/31346015-ec930004-ad17-11e7-9b19-f6f73d801069.png)

Needs https://github.com/Yoast/yoast-components/pull/346

Fixes #8012
